### PR TITLE
Update menu options

### PR DESCRIPTION
**Pull request by get-itips**
Closes https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/2855

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/mailbox-migration/use-minimal-hybrid-to-quickly-migrate.md [hunk 1]
```diff
@@ -76,25 +76,27 @@ During the migration, the Simple Mail Transfer Protocol (SMTP) address of each o
...
-    ![Go to Users \> Data migration in the admin center.](media/95477fe8-d213-462d-bd77-a00e2354234b.png)
+2. On the **Migrations** page, choose **Email**.
 
-2. **Migration** page, under **Select your data service**, choose **Exchange**.
+3. On the **Migrate your email** page, choose **Get started**.
+ 
+4. On the **Email sources** page, choose **Download the Hybrid Configuration Wizard**.
 
-    ![On the Migration page select Exchange as your data service.](media/f8fe6289-9aa6-4bbe-9a8f-0f424405c0e6.png)
+5. On the Download and run the **Office 365 Hybrid Configuration Wizard** page, choose **Download application**
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mailbox-migration/use-minimal-hybrid-to-quickly-migrate.md:87]:
```suggestion
5. On the **Download and run the Office 365 Hybrid Configuration Wizard** page, choose **Download application**.
```

File: Exchange/ExchangeOnline/mailbox-migration/use-minimal-hybrid-to-quickly-migrate.md [hunk 2]
```diff
@@ -76,25 +76,27 @@ During the migration, the Simple Mail Transfer Protocol (SMTP) address of each o
...
 
-3. On the first **Hybrid Configuration Wizard** page, choose **next** and on the **On-premises Exchange Server Organization** page, accept the default values and choose **next**.
+6. On the first **Hybrid Configuration Wizard** page, choose **next** and on the **On-premises Exchange Server Organization** page, accept the default values and choose **next**.
 
     By default the wizard connects to the Exchange server running the latest version.
 
-4. On the Credentials page, choose **Use current Windows credentials** for on-premises Exchange server, and enter admin credentials for it and your Microsoft 365 or Office 365 organization choose **next**, and then choose **next** again once the connections and credentials have validated.
+7. On the Credentials page, choose **Use current Windows credentials** for on-premises Exchange server, and enter admin credentials for it and your Microsoft 365 or Office 365 organization choose **next**, and then choose **next** again once the connections and credentials have validated.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mailbox-migration/use-minimal-hybrid-to-quickly-migrate.md:93]:
```suggestion
7. On the **Credentials** page, choose **Use current Windows credentials** for on-premises Exchange server, and enter admin credentials for it and your Microsoft 365 or Office 365 organization. Choose **next**, and then choose **next** again once the connections and credentials have validated.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review.

**Review by scanum — APPROVED** [review]

**Comment by scanum** [comment]
@tiburd PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**